dlp/snippets/deid/deid_test.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package deid contains example snippets using the DLP deidentification API.
+package deid
+
+import (
+	"bytes"
+	"context"
+	"log"
+	"os"
+	"testing"
+
+	dlp "cloud.google.com/go/dlp/apiv2"
+	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
+)
+
+var client *dlp.Client
+var projectID string
+
+func TestMain(m *testing.M) {
```

Comment:
Won't need this once functions initialize their own client. Also, tests should probably just use `tc.ProjectID`.

Reply:
Done